axelrod/strategies/graaskamp.py

```diff
@@ -0,0 +1,86 @@
+import random
+from axelrod.action import Action
+from axelrod.player import Player
+from scipy.stats import chisquare
+
+C, D = Action.C, Action.D
+
+
+class Graaskamp(Player):
+    """
```

Comment:
Could you take a look at this documentation about the documentation: https://axelrod.readthedocs.io/en/stable/tutorials/contributing/strategy/docstrings.html

We're missing a reference (see the other strategies in https://github.com/Axelrod-Python/Axelrod/blob/master/axelrod/strategies/axelrod_first.py